.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
-node_js: stable
+node_js:
+  - '14.15.4'
```

Comment:
let's change this to circle in a next PR?

Reply:
actually I revert to a LTS, stable currently mean 15.0 which is not long time supported

Reply:
sorry, I meant changing from travis to circleci

Reply:
ah yes indeed !